modules/claims_api/app/controllers/claims_api/v2/blueprints/intent_to_file_blueprint.rb

```diff
@@ -12,7 +12,7 @@ class IntentToFileBlueprint < Blueprinter::Base
           {
             creation_date: itf[:creation_date],
             expiration_date: itf[:expiration_date],
-            type: ClaimsApi::V2::IntentToFile::ITF_TYPES_TO_BGS_TYPES.index(itf[:type]),
+            type: ClaimsApi::V2::IntentToFile::ITF_TYPES_TO_BGS_TYPES[itf[:type]],
```

Comment:
`itf[:type]` is "compensation" and `ClaimsApi::V2::IntentToFile::ITF_TYPES_TO_BGS_TYPES` is a hash which looks like this:

```ruby
{"compensation"=>"C", "survivor"=>"S", "pension"=>"P"}
```

If you want to get a result using `Hash#index`, you'd need to pass "C", not "compensation". Passing `itf[:type]` to `index` here always yields `nil`.

It looks like the change you've made now reflects the intent of the original author. 👍